src/components/scenes/GuiPluginListScene.tsx

```diff
@@ -442,25 +448,44 @@ const getStyles = cacheStyles((theme: Theme) => ({
   }
 }))
 
-export const GuiPluginListScene = connect<StateProps, DispatchProps, OwnProps>(
-  (state, props) => {
-    const context = state.core.context
-    const deviceId = base58ToUuid(context.clientId)
-    const direction = props.route.name === 'pluginListSell' ? 'sell' : 'buy'
-    return {
-      account: state.core.account,
-      accountPlugins: state.account.referralCache.accountPlugins,
-      accountReferral: state.account.accountReferral,
-      coreDisklet: state.core.disklet,
-      countryCode: state.ui.settings.countryCode,
-      developerModeOn: state.ui.settings.developerModeOn,
-      deviceId,
-      disablePlugins: state.ui.exchangeInfo[direction].disablePlugins
-    }
-  },
-  dispatch => ({
-    updateCountryCode(countryCode: string) {
-      dispatch(updateOneSetting({ countryCode }))
-    }
-  })
-)(withTheme(GuiPluginList))
+export const GuiPluginListScene = React.memo((props: OwnProps) => {
+  const { navigation, route } = props
+  const dispatch = useDispatch()
+  const theme = useTheme()
+
+  const account = useSelector(state => state.core.account)
+  const accountPlugins = useSelector(state => state.account.referralCache.accountPlugins)
+  const accountReferral = useSelector(state => state.account.accountReferral)
+  const deviceId = useSelector(state => base58ToUuid(state.core.context.clientId))
+  const coreDisklet = useSelector(state => state.core.disklet)
+  const countryCode = useSelector(state => state.ui.settings.countryCode)
+  const developerModeOn = useSelector(state => state.ui.settings.developerModeOn)
+  const direction = props.route.name === 'pluginListSell' ? 'sell' : 'buy'
+  const disablePlugins = useSelector(state => state.ui.exchangeInfo[direction].disablePlugins)
+
+  const updateCountryCode = (countryCode: string) => {
+    dispatch(updateOneSetting({ countryCode }))
+  }
+
+  return (
+    <SceneWrapper background="theme" hasTabs hasNotifications>
+      {gap => (
+        <GuiPluginList
+          navigation={navigation}
+          route={route}
+          deviceId={deviceId}
+          account={account}
+          accountPlugins={accountPlugins}
+          accountReferral={accountReferral}
+          coreDisklet={coreDisklet}
+          countryCode={countryCode}
+          developerModeOn={developerModeOn}
+          disablePlugins={disablePlugins}
+          updateCountryCode={updateCountryCode}
+          theme={theme}
+          contentContainerStyle={{ paddingBottom: gap.bottom }}
+        />
+      )}
```

Comment:
`style={ marginBottom: -gap.bottom }`

Reply:
No this is right, it should be positive. The notificationHeight is meant as an
contentContainerStyle paddingBottom (overscroll) - using it as a style padding causes a visual cutoff